examples/picking/mesh_picking.rs

```diff
@@ -0,0 +1,123 @@
+//! A simple 3D scene to demonstrate mesh picking.
```

Comment:
Uhhh so I did *not* mean to commit this here lmao, please hold off on merging until I remove this later today

Reply:
Fixed